infrastructure/.terraform-version

```diff
@@ -1 +1 @@
-0.13.4
+1.3.7
```

Comment:
As per my message I have just sent you need to jump each version you can't do this big of a jump.